tests/core/scenario/test_scenario.py

```diff
@@ -156,6 +163,29 @@ def test_create_scenario_and_add_sequences():
     assert scenario.sequences == {"sequence_1": scenario.sequence_1, "sequence_2": scenario.sequence_2}
 
 
+def test_get_set_property_and_attribute():
+    dn_cfg = Config.configure_data_node("bar")
+    s_cfg = Config.configure_scenario("foo", additional_data_node_configs=[dn_cfg], key="value")
+    scenario = create_scenario(s_cfg)
+
+    assert scenario.properties == {"key": "value"}
+    assert scenario.key == "value"
+
+    scenario.properties["new_key"] = "new_value"
+    scenario.another_key = "another_value"
+
+    assert scenario.key == "value"
+    assert scenario.new_key == "new_value"
+    assert scenario.another_key == "another_value"
```

Comment:
It feels strange to me that
You can read a property through `scenario.key == "value"`
but you cannot write it using `scenario.another_key  = "another_value"` 

I did not realize that it can be really confusing.

What do you think? Should we just not expose properties as attributes ?

Reply:
yep, I think we should not expose `properties`'s keys as attributes, will remove it from setattr and getattr